syntax = "proto3";

package C2S;


enum C2S_MSG_ID
{
	MSG_ID_NONE = 0;
	
	MSG_ID_HEARTBEAT = 10001;

	MSG_ID_LOGIN_REQ = 10002;
	MSG_ID_LOGIN_RES = 10003;
	
	MSG_ID_LOGIN_ZONE_REQ = 10004;
	MSG_ID_LOGIN_ZONE_RES = 10005;
}


message C2SHeartBeat
{
	int32 state = 1;
}

message C2SLoginReq
{
	string sdk_user_id = 1;
	string sdk_token = 2;
	int32 channel_id = 3;
}

message C2SLoginRes
{
	int32 error_code = 1;
	string zone_token = 2;
	string host = 3;
	int32 port = 4;
}


message C2SLoginZoneReq
{
	string user_id = 1;
	string zone_token = 2;
}

message C2SLoginZoneRes
{
	int32 error_code = 1;
}




